lib/elastic_apm/config.rb

```diff
@@ -246,9 +247,28 @@ def load_env
     end
 
     def build_logger
-      Logger.new(log_path == '-' ? $stdout : log_path).tap do |logger|
+      load_error = false
+      if self.log_ecs_formatting == 'override'
+        begin
+          require 'ecs_logging/logger'
+          return ::EcsLogging::Logger.new(log_path == '-' ? $stdout : log_path).tap do |logger|
+            logger.level = log_level
+          end
+        rescue LoadError
+          load_error = true
+        end
+      end
```

Comment:
I think we could make this easier to read for ourselves by splitting it up into a few smaller private methods instead.